src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -311,12 +404,27 @@ export default {
 		getFocusableList() {
 			return this.$el.querySelectorAll('li.acli_wrapper .acli')
 		},
+		setIsFocused(value) {
+			this.isFocused = value
+		},
+
+		handleFilter(filter) {
+			this.isFiltered = true
+			this.filterText = filter
```

Comment:
```suggestion
			this.isFiltered = filter
```
Maybe just leave one property, with optional values `['mentions' , 'unread', null]`?